src/pages/MemberSettingsPage/ChangePassword/index.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
+import { Box, Typography, TextField, Button } from "@mui/material";
+import * as yup from "yup";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { useNavigate } from "react-router-dom";
+
+const ChangePasswordPage = () => {
+  const [currentPassword, setCurrentPassword] = useState("");
+  const [newPassword, setNewPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const [errors, setErrors] = useState({});
+
+  const navigate = useNavigate();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    schema
+      .validate(
+        { currentPassword, newPassword, confirmPassword },
+        { abortEarly: false }
+      )
+      .then(() => {
+        // Submit form
+      })
+      .catch((err) => {
+        const newErrors = {};
+        err.inner.forEach((error) => {
+          newErrors[error.path] = error.message;
+        });
+        setErrors(newErrors);
+      });
+  };
+
+  const handleCurrentPasswordChange = (event) => {
+    setCurrentPassword(event.target.value);
+  };
+
+  const handleNewPasswordChange = (event) => {
+    setNewPassword(event.target.value);
+  };
+
+  const handleConfirmPasswordChange = (event) => {
+    setConfirmPassword(event.target.value);
+  };
```

Comment:
Lets use our current library Formik for the state changes with the forms

Reply:
update made